CMakeLists.txt

```diff
@@ -977,10 +977,78 @@ add_library_variant(
     RAM_SIZE 2M
     STACK_SIZE 4K
 )
+add_library_variant(
+    armv7a
+    SUFFIX soft_nofp
+    COMPILE_FLAGS "-mfloat-abi=soft -march=armv7-a+nofp"
+    MULTILIB_FLAGS "--target=thumbv7-none-unknown-eabi -mfpu=none"
```

Comment:
I'm pretty sure this is incorrect. Please add tests in `test/multilib` to ensure these work and continue working.

Reply:
You are right.
I added tests and fixed the flags. Thanks.